evtwebsocket.go

```diff
@@ -104,15 +104,15 @@ func (c *Conn) onMsg(msg []byte) {
 	}
 	// If we didn't find a propper callback we
 	// just fire the OnMessage global handler
-	go c.OnMessage(msg)
+	go c.OnMessage(msg, c)
 }
 
 func (c *Conn) close() {
 	c.ws.Close()
 	c.closed = true
 	if c.Reconnect {
 		for {
-			if err := c.Dial(c.url); err == nil {
+			if err := c.Dial(c.url, ""); err == nil {
```

Comment:
It seems we would like to save the subprotocol as a parameter of Conn so we can pass it here on reconnect